subprojects/docs/src/docs/userguide/migration/upgrading_version_7.adoc

```diff
@@ -131,6 +131,15 @@ When you are using a `@link:{groovyDslPath}/org.gradle.api.tasks.Copy.html[Copy]
 [[changes_7.2]]
 == Upgrading from 7.1 and earlier
 
+[[incremental_task_inputs_deprecation]]
+=== IncrementalTaskInputs type is deprecated
+
+The `IncrementalTaskInputs` type was used to implement _incremental tasks_, that is to say tasks which can be optimized to run on a subset of changed inputs instead of the whole input.
```

Comment:
Very nit, but this is typographically preferrable:

```suggestion
The `IncrementalTaskInputs` type was used to implement _incremental tasks,_ that is to say tasks that can be optimized to run on a subset of changed inputs instead of the whole input.
```